nixos/modules/hardware/video/nvidia.nix

```diff
@@ -238,6 +244,11 @@ in {
           assertion = cfg.open -> (cfg.package ? open && cfg.package ? firmware);
           message = "This version of NVIDIA driver does not provide a corresponding opensource kernel driver";
         }
+
+        {
```

Comment:
Had to look through some of my own WIP code, but powerd is available starting from version `510.39.01`

Reply:
If it is, it's not documented
[510.39.01](https://download.nvidia.com/XFree86/Linux-x86_64/510.39.01/README/dynamicboost.html)
[510.47.04](https://download.nvidia.com/XFree86/Linux-x86_64/510.47.03/README/dynamicboost.html)

Reply:
From `NVIDIA_Changelog`:

     2022-01-11 version 510.39.01

    * Added a new daemon, nvidia-powerd, to provide support for
      the Dynamic Boost feature on supported systems. Dynamic Boost
      balances power between the CPU and the GPU for improved performance.
      For more details, see the "Dynamic Boost on Linux" chapter in the README.

Looks like they added the files, but didn't actually include the documentation for that release. I guess its fine to leave on the original version then.